<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstruirDialog</class>
 <widget class="QDialog" name="ConstruirDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Construccion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>62</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Casitas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Propiedades:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPropiedades">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Casitas a Construir:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="casitas">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Construir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>62</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Hoteles</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Propiedades:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPropiedadesHotel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Poner Hotel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio Unitario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrecioCasitas">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrecioHotel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio Unitario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Dinero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDinero">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
